overlord/devicestate/handlers_install.go

```diff
@@ -1284,6 +1286,174 @@ func (m *DeviceManager) loadAndMountSystemLabelSnaps(systemLabel string) (
 	return sys, snapInfos, snapSeeds, mntPtForType, unmount, nil
 }
 
+func structureFromName(name string, Structures []gadget.VolumeStructure) (*gadget.VolumeStructure, error) {
+	var foundStruct *gadget.VolumeStructure
+	for _, structure := range Structures {
+		// Future go versions will have per-iter variables... but not 1.13
+		structure := structure
```

Comment:
This is a bit too subtle IMHO, I would keep the comment and expand it to explain how the code should change after we require a more recent go version, but meanwhile use a different variable name here.